tests/unit/s2n_tls13_keys_test.c

```diff
@@ -296,13 +297,32 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_psk_new_secret(&test_psk, resumption_secret.data, resumption_secret.size));
 
         DEFER_CLEANUP(struct s2n_tls13_keys test_keys, s2n_tls13_keys_free);
-        GUARD(s2n_tls13_keys_init(&test_keys, test_psk.hmac_alg));
+        EXPECT_SUCCESS(s2n_tls13_keys_init(&test_keys, test_psk.hmac_alg));
 
         EXPECT_SUCCESS(s2n_tls13_derive_binder_key(&test_keys, &test_psk));
 
         S2N_BLOB_EXPECT_EQUAL(test_keys.extract_secret, expected_resumption_early_secret);
         S2N_BLOB_EXPECT_EQUAL(test_keys.derive_secret, expected_binder_key);
     }
 
+    /* Tests s2n_tls13_derive_early_secrets produces the correct secret when a psk is set. */
+    {
+        S2N_BLOB_FROM_HEX(resumption_early_secret,
+            "9b2188e9b2fc6d64d71dc329900e20bb41915000f678aa839cbb797cb7d8332c");
+        S2N_BLOB_FROM_HEX(expected_derived_secret,
+            "5f1790bbd82c5e7d376ed2e1e52f8e6038c9346db61b43be9a52f77ef3998e80");
```

Comment:
Could you add a comment on where/how these values were obtained? This will come handy if we have to update it in the future.